### ISSUE TITLE
consent form emailed to participants for record keeping

### ISSUE THREAD

**Issue by elainewolfe** [qiime/qiime_web_app#628]
A lot of participants are replying to the automated email sent out to give them a copy of the consent form. Maybe we could add something either in the subject line or the body saying "THIS IS A COPY OF THE CONSENT FORM FOR YOUR RECORDS". That might reduce confusion.

**Comment by ElDeveloper** [comment]
:+1:

Yoshiki Vázquez-Baeza

**Comment by adamrp** [comment]
I agree -- this has caused confusion for several participants in the past.
 Clearly, people that are confused by this are people who are not actually
reading the consent form that they're signing electronically, but whatever.
 I'm all for changing the subject line (which I hope they actually
_are_ reading)
to be more explicit...
